docs/reference/platforms/index.rst

```diff
@@ -23,48 +23,60 @@ Platform support
 +-----+-------------------------------------+
 
 
-.. |Gradle| replace:: **Gradle**
+.. |Gradle| replace:: **Gradle project**
 .. _Gradle: ./android/gradle.html
 
-.. |iOS| replace:: **Xcode**
+.. |iOS| replace:: **Xcode project**
 .. _iOS: ./iOS/xcode.html
 
-.. |System| replace:: **Native System Packages**
+.. |AppImage| replace:: **AppImage**
+.. _AppImage: ./linux/appimage.html
+
+.. |Flatpak| replace:: **Flatpak**
+.. _Flatpak: ./linux/flatpak.html
```

Comment:
So - the reason these replace calls exist at all is so that we can make the headings bold inside a link. The Bold was intended to indicate the platform default. As long as you don't need the bold, you don't *strictly* need the replace, either.

I've tweaked to retain the replace, but remove the bolds.

Reply:
Ahhh, I see; the **bold** represents defaults. That apparently never clicked with me. I thought the differences in appearance in the table were being driving by the differences in how the links were constructed....and I rationalized that the link constructions were different because of some sphinx/rest thing I didn't understand....

Well, I may not have changed anything if I knew that before 😆